build-kernel.sh

```diff
@@ -72,24 +75,36 @@ if [[ -e /patches-grsec && -n "$GRSECURITY" && "$GRSECURITY" = "1" ]]; then
     find /patches-grsec -maxdepth 1 -type f -exec patch -p 1 -i {} \;
 fi
 
-echo "Copying in our mkdebian"
-cp /usr/local/bin/mkdebian scripts/package/mkdebian
+# Generate the orig tarball
+#tar --use-compress-program="xz -T 0" -cf ../linux-upstream_${LINUX_VERSION}-grsec-${LOCALVERSION}.orig.tar.xz .
+tar -cf - . | pigz > ../linux-upstream_${LINUX_VERSION}-grsec-${LOCALVERSION}.orig.tar.gz
 
+echo "Copying in our debian/"
+cp -R /debian debian
+
+export PACKAGE_VERSION="${LINUX_VERSION}-grsec-${LOCALVERSION}-1"
+export DEBARCH="amd64"
+
+cat debian/control.in | envsubst > debian/control
+echo "" >> debian/control
 if [[ "$LOCALVERSION" = "-workstation" ]]; then
-    echo "Copying in our securedrop-workstation-grsec"
-    mkdir -p debian/securedrop-workstation-grsec
-    cp -Rv /securedrop-workstation-grsec/* debian/securedrop-workstation-grsec/
+    echo "Generating d/control for workstation"
+    cat debian/control.workstation | envsubst >> debian/control
 else
-    echo "Copying in our securedrop-grsec"
-    mkdir -p debian/securedrop-grsec
-    cp -Rv /securedrop-grsec/* debian/securedrop-grsec/
+    echo "Generating d/control for server"
+    cat debian/control.server | envsubst >> debian/control
 fi
+cat debian/changelog.in | envsubst > debian/changelog
+
+cat <<EOF > debian/rules.vars
+ARCH := x86
+KERNELRELEASE := ${LINUX_VERSION}
```

Comment:
I think we also need to add our version stuff here, because I think `uname -a`, et al. don't have the correct version.